### PR TITLE
(Mailables): Support Simultaneous Small and Large Print

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

The mailables large print feature currently disables the small print on the resulting pdf. This PR corrects this, allowing for simultaneous large and small prints. Unfortunately, there is no way to change the count for the second size. Not sure how best to solve this.

**PR Checklist:**
- [no] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [no] Are all languages supported (Internationalization/Localization)?
- [no] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]
lgtm

File: lib/util/mailables.js [hunk 1]
```diff
@@ -113,13 +113,22 @@ async function writePDF(formData, imageData, otpConfig) {
...
+          rows.push([name, quantity])
+        }
+
+        if (largePrint && largeFormat) {
+          name += ' (LARGE PRINT)'
+          rows.push([name, quantity])
+        }
+        return rows
+      })
+      .flat()
```
Comment (amy-corson-ibigroup) [inline lib/util/mailables.js:131]: Nitpicky, but could just change the above `map` to `flatMap`

**Review by amy-corson-ibigroup — COMMENTED** [review]
Could you get rid of the checkboxes and just have two inputs? One for small format one for large? 
Having trouble understanding why you'd want to select both checkboxes without being able to control the number for each, but it it makes sense in context I trust you

File: lib/util/mailables.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/util/mailables.js:131]: This is much cleaner thanks!

**Pull request merged**